src/routes/(main)/tournaments/[id]/edit/+page.svelte

```diff
@@ -35,6 +36,13 @@
 
 <svelte:head>
 	<title>Edit Tournament - {name}</title>
+
+	<meta name="twitter:card" content="summary" />
+	<meta property="og:title" content="{name} Settings" />
+	<meta property="og:type" content="website" />
+	<meta property="og:description" content="Configure main tournament settings (host only)" />
+	<meta property="og:image" content={banner_url} />
+	<meta name="theme-color" content={color} />
 </svelte:head>
```

Comment:
Social sites like discord ping the server directly to fetch open graph information, and since this endpoint does not have public permissions, this embed will never show.

You can remove the open-graph information on the endpoints that cannot be publicly viewed.